flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/snapshot/RocksIncrementalSnapshotStrategy.java

```diff
@@ -209,7 +209,7 @@ private SnapshotDirectory prepareLocalSnapshotDirectory(long checkpointId) throw
 			}
 		} else {
 			// create a "temporary" snapshot directory because local recovery is inactive.
-			Path path = new Path(instanceBasePath.getAbsolutePath(), "chk-" + checkpointId);
+			Path path = new Path(instanceBasePath.toURI().toString(), "chk-" + checkpointId);
```

Comment:
Just a trivial modification, how about change this line to keep symmetrical to local-recovery-enabled code?
~~~ java
				File snapshotDir = new File(instanceBasePath, "chk-" + checkpointId);
				Path path = new Path(snapshotDir.toURI());
~~~

Reply:
Yes, that would look better. Thanks for your review.